beacon-chain/p2p/gossip_scoring_params.go

```diff
@@ -273,6 +314,21 @@ func scoreDecay(totalDurationDecay time.Duration) float64 {
 	return math.Pow(decayToZero, 1/float64(numOfTimes))
 }
 
+func decayThreshold(decayRate, rate float64) (float64, error) {
+	d, err := decay(decayRate, rate)
+	if err != nil {
+		return 0, err
+	}
+	return d * decayRate, nil
+}
+
+func decay(decayRate, rate float64) (float64, error) {
```

Comment:
Here as well, can benefit from function description, as normally, I'd expect to see `value * decoyRate` (where `0 < decayRate < 1`) returned. But since it does calculate things differently (rather `decayThreshold()` does what I expect and `decay()` has to do with rate), descriptive comment will be really useful.